---
default_fault: 0
default_state: 0
default_joint: 0
default_time: 0
default_time_label: 0
default_time_offset: 0.5
min_time_offset: 0.5
max_time_offset: 2
max_time: 3
max_joints: 8
max_faults: 4
max_states: 9
max_time_labels: 3